src/lib/updating-element.ts

```diff
@@ -723,10 +723,10 @@ export abstract class UpdatingElement extends HTMLElement {
     if (result != null) {
       await result;
     }
-    return !this._hasRequestedUpdate;
+    return !this.isUpdatePending;
   }
 
-  private get _hasRequestedUpdate() {
+  protected get isUpdatePending() {
```

Comment:
Why is it necessary to expose this (what use case is unlocked)?

Reply:
No clear use case. Removing for now.